source/transfer_list.rst

```diff
@@ -343,6 +343,23 @@ Inputs:
 
    #. Add `align8(te.hdr_size + te.data_size)` to `te_base_addr`.
 
+
+Adding a void TE
+^^^^^^^^^^^^^^^^
+
+Inputs:
+
+- `te_base_addr`: Base address where void TE to be added
+- `size`: Total size available for void TE
+
```

Comment:
You also need to xor the checksum here, to first remove the old state of those bytes from the checksum before adding the new one.